### PR TITLE
Milestone: Feature Create a database based on a diagram

### PR DESCRIPTION
**Pull request by darikmohammed**
# Features

- Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Follow the schema to create the database. 

Screen Shot 2022-08-17 at 4 11 02 PM

- Join tables from many-to-many relationships.
- Add the FK indexes

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    PRIMARY KEY(ID),
+
+    CONSTRAINT FK_MEDICAL_HISTORY_ID 
+    FOREIGN KEY (MEDICAL_HISTORIES_ID)
+    REFERENCES MEDICAL_HISTORIES(ID),
+
+    CONSTRAINT FK_TREATMENTS_ID 
+    FOREIGN KEY (TREATMENT_ID)
+    REFERENCES TREATMENTS(ID)
+);
```
Comment (vechicin) [inline schema_based_on_diagram.sql]: Please create indexes for all your foreign keys.

Screen Shot 2022-08-17 at 8 19 39

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @darikmohammed and @kiboma2021 ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job creating the database schema!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (darikmohammed) [reply schema_based_on_diagram.sql]: Thanks we Forgot to add it. Fixed now

**Review by Munsa1 — APPROVED** [review]
Hi @darikmohammed, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**